### ISSUE TITLE
Material You dynamic colors support

### ISSUE THREAD

**Issue by BigPebble123** [mollyim/mollyim-android#300]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature description

Hi! 👋🏻

Could you please implement this?
https://github.com/nekorubu/Signal-You

I know that someone [already created a feature request](https://github.com/mollyim/mollyim-android/issues/112) for Marerial You upport and @valldrac replied that Signal's team is already working on it, but that was in 2022 (almost 4 years ago).

Recently @nekorubu has created his own fork that adopts Material You dynamic theming and it would be great to implement it in Molly.

**Comment by hndrk-themer** [comment]
True, this would be a great addition to create a way more consistent look and imho, not that hard to achieve (am using an overlay rn on Molly, but stock for all people would be the best ofc).

Basic Info:
https://m3.material.io/styles/color/system/how-the-system-works

The Figma Material Theme Builder Plugin is very useful for testing (set to Dynamic):
https://www.figma.com/community/plugin/1034969338659738588/material-theme-builder

Basic M3 color attributes:
https://github.com/material-components/material-components-android/blob/master/docs/theming/Color.md

**Comment by katherine-maxwell** [comment]
I would also quite like this dynamic theming. I wanted to note that the particular colors chosen in the nekorubu Signal-You fork could use some adjusting for better contrast. In particular, the bottom nav bar has poor contrast. Hopefully Molly can have an improved Material You implementation.

**Comment by nekorubu** [comment]
> I wanted to note that the particular colors chosen in the nekorubu Signal-You fork could use some adjusting for better contrast. In particular, the bottom nav bar has poor contrast. Hopefully Molly can have an improved Material You implementation.

I'd like to think that's because I'm using Repainter for my Material You colors, and with the adjustments I made, the neutral colors don't have as much contrast. I did try to stick with the Material You guidelines as much as I could, though

**Comment by katherine-maxwell** [comment]
> > I wanted to note that the particular colors chosen in the nekorubu Signal-You fork could use some adjusting for better contrast. In particular, the bottom nav bar has poor contrast. Hopefully Molly can have an improved Material You implementation.
> 
> I'd like to think that's because I'm using Repainter for my Material You colors, and with the adjustments I made, the neutral colors don't have as much contrast. I did try to stick with the Material You guidelines as much as I could, though

To explain with pictures, using the same color theme on my phone for all screenshots:

Signal-You appearance:
![signal-2024-03-26-23-39-49-335-4](https://github.com/mollyim/mollyim-android/assets/165039804/ff4c8985-3509-41c1-89b8-f05328bb7ab7)

Clock-You appearance:
![signal-2024-03-26-23-39-49-335-5](https://github.com/mollyim/mollyim-android/assets/165039804/b010b736-96a6-4257-ab0b-0fcae513ab05)

However, some apps appear differently, example is Jerboa (using Repainter to select Rainbow and Vibrant variants):
![signal-2024-03-26-23-39-49-335-2](https://github.com/mollyim/mollyim-android/assets/165039804/4cb48674-c310-4ce9-9353-cffe68b67bfe)
![signal-2024-03-26-23-39-49-335-1](https://github.com/mollyim/mollyim-android/assets/165039804/4663d985-c7f8-4b17-b311-3cc718e727ca)

**Comment by hndrk-themer** [comment]
If it's getting implemented right, i would definetely stick to the material 3 guidelines. so it matches the whole material 3 ecosystem. everything else can get customized by the user.

**Comment by nekorubu** [comment]
> To explain with pictures, using the same color theme on my phone for all screenshots:
> 
> Signal-You appearance: ![signal-2024-03-26-23-39-49-335-4](https://private-user-images.githubusercontent.com/165039804/316909418-ff4c8985-3509-41c1-89b8-f05328bb7ab7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTA5NDE4LWZmNGM4OTg1LTM1MDktNDFjMS04OWI4LWYwNTMyOGJiN2FiNy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZGEyMzVhMmVlZTUyNmY3MDJiMjljOTljZDNiNDEwZGU5NTljY2NlZTAzNTQxNjRkM2YwMzg2NDYwMWY2ZWJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yIo15C-aJA5cyVGlZY-jhyuaQNAT9cK1fUW_WM9DT44)
> 
> Clock-You appearance: ![signal-2024-03-26-23-39-49-335-5](https://private-user-images.githubusercontent.com/165039804/316909636-b010b736-96a6-4257-ab0b-0fcae513ab05.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTA5NjM2LWIwMTBiNzM2LTk2YTYtNDI1Ny1hYjBiLTBmY2FlNTEzYWIwNS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYWJmYTAxYmFjNjQ4N2U3OTBlMzU1MWJhMDMxODVhY2MyODE1M2E1OWM4ODhkOWNmYjU0NDRjZmNhOWEwODM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W4tKH3_s13qMoXamsJ-R18indea1HGXx9tzezqSECWg)
> 
> However, some apps appear differently, example is Jerboa (using ColorBlendr to select Rainbow and Vibrant variants): ![signal-2024-03-26-23-39-49-335-2](https://private-user-images.githubusercontent.com/165039804/316910684-4cb48674-c310-4ce9-9353-cffe68b67bfe.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTEwNjg0LTRjYjQ4Njc0LWMzMTAtNGNlOS05MzUzLWNmZmU2OGI2N2JmZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODJiZTY5NjU3YjNhMDU3ZmVlMTBmMDU5Yjk5MDZiZGRkZTZiMDZjY2NlMDgxMDBmMmU5Y2YzOTM0MTEwZjNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nayTDenH0RZYrV-Xh3RUipDjpfuTLeco74Qdwua6hlo) ![signal-2024-03-26-23-39-49-335-1](https://private-user-images.githubusercontent.com/165039804/316910743-4663d985-c7f8-4b17-b311-3cc718e727ca.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTEwNzQzLTQ2NjNkOTg1LWM3ZjgtNGIxNy1iMzExLTNjYzcxOGU3MjdjYS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02N2FhNjkzMDIzMGFjZTA5NDEzNGFkMDc2NTA2NzNiNTdmN2Q4MWI4ZjI5OWY4OTcyOGZmYmQyYjRkYzU0NjlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QXh8E-npQMLlTrcZiX2xsE5q7YeJ7T7cRVwqzgDbQnI)

Ohhh, I see now. I'll try to find some time to change the colors on my fork. Thanks for the feedback!

**Comment by Fernan181** [comment]
Any plans to get a pr for this? I would really love to see this feature implemented into Molly.

**Comment by nekorubu** [comment]
> > To explain with pictures, using the same color theme on my phone for all screenshots:
> > Signal-You appearance: ![signal-2024-03-26-23-39-49-335-4](https://private-user-images.githubusercontent.com/165039804/316909418-ff4c8985-3509-41c1-89b8-f05328bb7ab7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTA5NDE4LWZmNGM4OTg1LTM1MDktNDFjMS04OWI4LWYwNTMyOGJiN2FiNy5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZGEyMzVhMmVlZTUyNmY3MDJiMjljOTljZDNiNDEwZGU5NTljY2NlZTAzNTQxNjRkM2YwMzg2NDYwMWY2ZWJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.yIo15C-aJA5cyVGlZY-jhyuaQNAT9cK1fUW_WM9DT44)
> > Clock-You appearance: ![signal-2024-03-26-23-39-49-335-5](https://private-user-images.githubusercontent.com/165039804/316909636-b010b736-96a6-4257-ab0b-0fcae513ab05.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTA5NjM2LWIwMTBiNzM2LTk2YTYtNDI1Ny1hYjBiLTBmY2FlNTEzYWIwNS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYWJmYTAxYmFjNjQ4N2U3OTBlMzU1MWJhMDMxODVhY2MyODE1M2E1OWM4ODhkOWNmYjU0NDRjZmNhOWEwODM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W4tKH3_s13qMoXamsJ-R18indea1HGXx9tzezqSECWg)
> > However, some apps appear differently, example is Jerboa (using ColorBlendr to select Rainbow and Vibrant variants): ![signal-2024-03-26-23-39-49-335-2](https://private-user-images.githubusercontent.com/165039804/316910684-4cb48674-c310-4ce9-9353-cffe68b67bfe.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTEwNjg0LTRjYjQ4Njc0LWMzMTAtNGNlOS05MzUzLWNmZmU2OGI2N2JmZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zODJiZTY5NjU3YjNhMDU3ZmVlMTBmMDU5Yjk5MDZiZGRkZTZiMDZjY2NlMDgxMDBmMmU5Y2YzOTM0MTEwZjNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.nayTDenH0RZYrV-Xh3RUipDjpfuTLeco74Qdwua6hlo) ![signal-2024-03-26-23-39-49-335-1](https://private-user-images.githubusercontent.com/165039804/316910743-4663d985-c7f8-4b17-b311-3cc718e727ca.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0NjcwNDMsIm5iZiI6MTcxMTQ2Njc0MywicGF0aCI6Ii8xNjUwMzk4MDQvMzE2OTEwNzQzLTQ2NjNkOTg1LWM3ZjgtNGIxNy1iMzExLTNjYzcxOGU3MjdjYS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMyNlQxNTI1NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02N2FhNjkzMDIzMGFjZTA5NDEzNGFkMDc2NTA2NzNiNTdmN2Q4MWI4ZjI5OWY4OTcyOGZmYmQyYjRkYzU0NjlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QXh8E-npQMLlTrcZiX2xsE5q7YeJ7T7cRVwqzgDbQnI)
> 
> Ohhh, I see now. I'll try to find some time to change the colors on my fork. Thanks for the feedback!

I finally implemented this in [the most recent release](https://github.com/nekorubu/Signal-You/releases/tag/v7.7.1) of Signal-You

**Comment by Fernan181** [comment]
@nekorubu great! Any chance we see a pr from you implementing your changes? I'd really appreciate it.

**Comment by katherine-maxwell** [comment]
@nekorubu  Thanks for your changes. While the bottom bar is clear now, the highlight for the active tab is still quite poor. This in particular bothers me, because I look for the highlight color to know which tab I am on.
![Screenshot_20240516-183300_1g](https://github.com/mollyim/mollyim-android/assets/165039804/df0bb333-5b13-4f4c-826d-846c68df9cc5)

From what I understand, there is a default material 3 color for elements. See https://m3.material.io/components/navigation-bar/specs. Also, I believe the elevation affects the color slightly, see https://m3.material.io/styles/elevation/tokens.

If/when I have free time I can look at the code myself and try to adjust it.

**Comment by nekorubu** [comment]
> @nekorubu Thanks for your changes. While the bottom bar is clear now, the highlight for the active tab is still quite poor. This in particular bothers me, because I look for the highlight color to know which tab I am on. ![Screenshot_20240516-183300_1g](https://private-user-images.githubusercontent.com/165039804/331142153-df0bb333-5b13-4f4c-826d-846c68df9cc5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU4NjAxMDksIm5iZiI6MTcxNTg1OTgwOSwicGF0aCI6Ii8xNjUwMzk4MDQvMzMxMTQyMTUzLWRmMGJiMzMzLTViMTMtNGY0Yy04MjZkLTg0NmM2OGRmOWNjNS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxNlQxMTQzMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NjdjZWJlMzNmYTI4MmM1YTAxOTllNDJhMmU1MjA2YWM3ZWYyNGVlYzc4MDliMWQ5YjQ3ZmQyMDJjODUzZjY5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.H-1yritnVWFz1qRcLb4N1RnAY_xnNRJqPE2oXBhF49I)
> 
> From what I understand, there is a default material 3 color for elements. See https://m3.material.io/components/navigation-bar/specs. Also, I believe the elevation affects the color slightly, see https://m3.material.io/styles/elevation/tokens.
> 
> If/when I have free time I can look at the code myself and try to adjust it.

Oh, I see what you mean. When I made the change, I mainly just eyeballed it until it looks right, which is quite ironic given that when I last made changes, I tried to go off of the guidelines. On top of that, I was using Repainter for my colors, so while the colors looked fine, now that I'm using colors that stick to the Material You spec, I can see where the contrast is still lacking.
Thanks again! I'm running a build through GitHub to hopefully get a test version now.

**Comment by nekorubu** [comment]
> @nekorubu great! Any chance we see a pr from you implementing your changes? I'd really appreciate it.

I'll try when I get the chance. I don't know if there's a difference in how Molly implements the colors in the code, so I'll have to check it out myself.

**Comment by resucutie** [comment]
hello! any updates on implementing material you on molly? we're all interested in this feautre being implemented

**Comment by nekorubu** [comment]
> @nekorubu Hi there! I've noticed that you've been keeping your repository up to date, which is great. However, I wanted to see if you have any plans to add M3 support for Molly? No rush at all - just curious since it's been a while since you said you will see if you could do it. Either way, thanks for your hard work on this!

Hey, sorry about the late reply. I've kinda been all over the place in life as of late.

I did manage to get Material You colors working in Molly: [Here's the fork with the changes](https://github.com/nekorubu/Molly-You)

The only problem that I don't know how to get the colors working as an option, and I didn't want to just make a pull request implementing it since Molly already has its own theme. So I honestly just forgot about it 😅

If the devs are fine with that and know how to implement those changes, or are fine with Material You colors taking over the app completely, then I can go ahead and make the pull request.

**Comment by BHydden** [comment]
> ... I'm pinging you ...
> What do you think?

People like you shouldn't be allowed on the internet. I haven't seen something this obnoxious in years.

**Comment by valldrac** [comment]
@BigPebble123 Please avoid @ mentioning all developers, especially those outside the Molly organization. We all receive notifications for comments here, so there's no need to ping everyone. Thanks!

**Comment by valldrac** [comment]
We’re open to accepting a pull request for this feature, but there are a couple of conditions:

- The implementation must include a toggle in the settings, allowing users to enable or disable the dynamic colors.
- Since Molly is a fork and we frequently pull changes from upstream, the theme could quickly become outdated. We’ll need someone committed to maintaining this theme over time to ensure it stays up to date.

If someone is willing to take the role of maintaining the theme, please go ahead and submit the pull request.

**Comment by valldrac** [comment]
@nekorubu We're releasing Material You as an alternate theme in Molly soon. We'd be very grateful if you could check it and share your thoughts. If you have any ideas to improve it, even better. We've also used the color palette you designed for Signal-You and want to make sure you get the credit for your work.

**Comment by hndrk-themer** [comment]
> We're releasing Material You as an alternate theme in Molly soon.

will there also be an option to theme chat-bg and bubble in material you as well, or "just" the rest of the UI like it happened before? am asking because the color picker is so limited that you cant really get a matching overall look rn.

Greetings and thanks for your work on this

**Comment by nekorubu** [comment]
@valldrac Hey! Sorry for the lack of response on my end. In terms of checking it, I take it I'll just need to clone the code and build it myself, or is there a release that I can install to check it? Looking at the code from the pull request, it looks really different from how I did it in Signal-You

**Comment by nekorubu** [comment]
Oh, okay, thanks! I just looked at it and left a rather long-winded bunch of comments in the room.

TL;DR: It's a bit different than how I did it, but I like it, and you guys got it to be more complete than on my end!

**Comment by Flatkat** [comment]
Then this issue should be closed.
Thank you to all the people involved, was looking forward a lot for this feature! ^.^
